### PR TITLE
Adding in Jenkins support for Coveralls

### PR DESCRIPTION
**Pull request by goosemo**
This change is [Reviewable](https://reviewable.io/reviews/coagulant/coveralls-python/107)

### CODE REVIEW

File: tests/test_api.py [hunk 1]
```diff
@@ -123,6 +123,12 @@ def test_circleci_no_config(self):
         assert cover.config['service_job_id'] == '888'
         assert cover.config['service_pull_request'] == '9999'
 
+    @patch.dict(os.environ, {'JENKINS_HOME': '/var/lib/jenkins', 'BUILD_NUMBER': '888'}, clear=True)
+    def test_circleci_no_config(self):
```
Comment (coagulant) [inline tests/test_api.py]: test_jenkins_no_config?

Comment (goosemo) [inline tests/test_api.py]: good catch, I'll update

**Comment by TheKevJames** [comment]
Hey there! This repo just underwent a change in maintainers -- myself and @goanpeca will be taking over this project. Give us some time to get up-to-speed and we'll make sure to take a look at this PR soon.

**Comment by TheKevJames** [comment]
Moved to #160

**Pull request closed without merging**